validator/src/main/java/com/amazon/aoc/validators/AbstractStructuredLogValidator.java

```diff
@@ -124,18 +119,23 @@ protected void fetchAndValidateLogs(Instant startTime) throws Exception {
 
   private void checkResult() throws BaseException {
     if (schemasToValidate.isEmpty() || schemasToValidate.keySet().equals(validatedSchema)) {
+      log.info(String.format("[StructuredLogValidator] log structure validation successful for %s"
```

Comment:
```suggestion
      log.info(String.format("[StructuredLogValidator] log structure validation successful"
```

Too many placeholders.